app/src/main/java/com/kickstarter/ui/activities/compose/projectpage/ConfirmPledgeDetailsScreen.kt

```diff
@@ -565,22 +605,40 @@ fun BonusSupportContainer(
                 Spacer(modifier = Modifier.width(dimensions.paddingMediumSmall))
             }
 
-            Text(
-                modifier = Modifier
-                    .background(
+            Row(
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier.background(
                         color = colors.kds_white,
                         shape = shapes.small
-                    )
-                    .padding(
-                        start = dimensions.paddingMediumSmall,
-                        top = dimensions.paddingMediumSmall,
-                        bottom = dimensions.paddingMediumSmall,
-                        end = dimensions.paddingMediumSmall
-                    ),
-                text = totalBonusAmount,
-                style = typography.headline,
-                color = colors.textAccentGreen
-            )
+                ).padding(
+                    start = dimensions.paddingXSmall,
+                    top = dimensions.paddingXSmall,
+                    bottom = dimensions.paddingXSmall,
+                    end = dimensions.paddingXSmall
+                ),
+
+            ) {
+                Text(
+                    text = currencySymbolAtStart ?: "",
+                    color = colors.kds_create_700
```

Comment:
@mtgriego KSColors tells me this is an old color, so pls let me know what I should be using instead.

Reply:
As this is text, it should probably be `textAccentGreen`, i believe thats the closest new green for text we have